testing/mod.ts

```diff
@@ -157,11 +158,8 @@ Object.assign(assertions.assert, assertions);
 
 export const assert = assertions.assert as Assert;
 
-/**
- * An alias to assert.equal
- * @deprecated
```

Comment:
Shouldn't `assert.equal = prettyAssert` and `assertEqual`  be removed?

Or at least, still be deprecated in the doc string.

Reply:
Right.
btw asserts will be refactored see: https://github.com/denoland/deno_std/pull/227

Reply:
But shouldn't this also do assert.equal = prettyAssert?

Reply:
mmmm you're right. done.
I've deleted the check of exception format from the mod_test because we already check the format in pretty_test and we ensure assert.equal == prettyAssert. So it's avoiding code duplicate for the format check.